scripts/release.py

```diff
@@ -144,23 +144,31 @@ def read_release_log(path=os.getcwd()):
                                        other_versions))
             msg_commit = f.readline().split("=")[1].strip()
             msg_tag = f.readline().split("=")[1].strip()
+            auto_increment_policy = f.readline().split("=")[1].strip()
         if other_versions == "::":
             other_versions = None
         return (remote_alias, branch, tag, next_snapshot,
                 maintenance_version, is_final, skipTests, skipITs, profiles,
-                other_versions, msg_commit, msg_tag)
+                other_versions, msg_commit, msg_tag, auto_increment_policy)
 
     # pylint: disable=R0913
     def __init__(self, repo, branch, tag, next_snapshot,
-                 maintenance_version="auto", is_final=False,
-                 skipTests=False, skipITs=False,
-                 other_versions=None, profiles='', msg_commit='', msg_tag=''):
+                 maintenance_version="auto",
+                 is_final=False,
+                 skipTests=False,
+                 skipITs=False,
+                 other_versions=None,
+                 profiles='',
+                 msg_commit='',
+                 msg_tag='',
+                 auto_increment_policy='auto_patch'):
```

Comment:
Won't we end up defining a dictionary with as much lines as the init params ?
Between:
```python
    def __init__(self, repo, branch, tag, next_snapshot,
                 maintenance_version="auto",
                 is_final=False,
                 skipTests=False,
                 skipITs=False,
                 other_versions=None,
                 profiles='',
                 msg_commit='',
                 msg_tag='',
                 auto_increment_policy='auto_patch'):
```
and 
```python
    def __init__(self, repo, branch, tag, next_snapshot, **kwargs):

        prop_defaults = {
            "maintenance_version": "auto",
            "is_final": False,
            "skipTests": False,
            "skipITs": False,
            "other_versions": None,
            "profiles": '',
            "msg_commit": '',
            "msg_tag": '',
            "auto_increment_policy": 'auto_patch',
        }

        for (prop, default) in prop_defaults.iteritems():
            setattr(self, prop, kwargs.get(prop, default))
```
I think we would really loose in readability.